tools/db_crashtest.py

```diff
@@ -776,6 +791,8 @@ def finalize_and_sanitize(src_params):
         # disableWAL and recycle_log_file_num options are not mutually
         # compatible at the moment
         dest_params["recycle_log_file_num"] = 0
+    if dest_params.get("block_align") == 1:
```

Comment:
Is there any reference for this adjustment, can you add some explanation for why this is needed here?

Reply:
Fixed